### PR TITLE
Fix curl examples in gateway docs

### PR DESCRIPTION
**Pull request by harupy**
## Related Issues/PRs

 #xxx

## What changes are proposed in this pull request?

Fix curl examples in gateway docs. `api/2.0/gateway` is missing.

## How is this patch tested?

- [x] Existing unit/integration tests
- [ ] New unit/integration tests
- [ ] Manual tests (describe details, including test results, below)

## Does this PR change the documentation?

- [ ] No. You can skip the rest of this section.
- [ ] Yes. Make sure the changed pages / sections render correctly in the documentation preview.

### Is this a user-facing change?

- [ ] No. You can skip the rest of this section.
- [ ] Yes. Give a description of this change to be included in the release notes for MLflow users.

(Details in 1-2 sentences. You can just refer to another PR with a description if this PR is part of a larger change.)

### What component(s), interfaces, languages, and integrations does this PR affect?

Components

- [ ] `area/artifacts`: Artifact stores and artifact logging
- [ ] `area/build`: Build and test infrastructure for MLflow
- [ ] `area/docs`: MLflow documentation pages
- [ ] `area/examples`: Example code
- [ ] `area/gateway`: AI Gateway service, Gateway client APIs, third-party Gateway integrations
- [ ] `area/model-registry`: Model Registry service, APIs, and the fluent client calls for Model Registry
- [ ] `area/models`: MLmodel format, model serialization/deserialization, flavors
- [ ] `area/recipes`: Recipes, Recipe APIs, Recipe configs, Recipe Templates
- [ ] `area/projects`: MLproject format, project running backends
- [ ] `area/scoring`: MLflow Model server, model deployment tools, Spark UDFs
- [ ] `area/server-infra`: MLflow Tracking server backend
- [ ] `area/tracking`: Tracking Service, tracking client APIs, autologging

Interface

- [ ] `area/uiux`: Front-end, user experience, plotting, JavaScript, JavaScript dev server
- [ ] `area/docker`: Docker use across MLflow's components, such as MLflow Projects and MLflow Models
- [ ] `area/sqlalchemy`: Use of SQLAlchemy in the Tracking Service or Model Registry
- [ ] `area/windows`: Windows support

Language

- [ ] `language/r`: R APIs and clients
- [ ] `language/java`: Java APIs and clients
- [ ] `language/new`: Proposals for new client languages

Integrations

- [ ] `integrations/azure`: Azure and Azure ML integrations
- [ ] `integrations/sagemaker`: SageMaker integrations
- [ ] `integrations/databricks`: Databricks integrations

<a name="release-note-category"></a>

### How should the PR be classified in the release notes? Choose one:

- [ ] `rn/breaking-change` - The PR will be mentioned in the "Breaking Changes" section
- [x] `rn/none` - No description will be included. The PR will be mentioned only by the PR number in the "Small Bugfixes and Documentation Updates" section
- [ ] `rn/feature` - A new user-facing feature worth mentioning in the release notes
- [ ] `rn/bug-fix` - A user-facing bug fix worth mentioning in the release notes
- [ ] `rn/documentation` - A user-facing documentation change worth mentioning in the release notes

### CODE REVIEW

File: docs/source/gateway/index.rst [hunk 1]
```diff
@@ -891,19 +891,25 @@ This endpoint allows you to submit a query to a configured provider route. The d
 
 .. code-block:: bash
 
-    curl -X POST http://my.gateway:8888/gateway/completions/invocations -H "Content-Type: application/json" -d '{"prompt": "Describe the probability distribution of the decay chain of U-235"}'
+    curl -X POST http://my.gateway:8888/api/2.0/gateway/gateway/completions/invocations \
```
Comment (dbczumar) [inline docs/source/gateway/index.rst:894]: Looks like gateway is duplicated

File: docs/source/gateway/index.rst [hunk 2]
```diff
@@ -891,19 +891,25 @@ This endpoint allows you to submit a query to a configured provider route. The d
...
+    curl -X POST http://my.gateway:8888/gateway/chat/invocations \
+      -H "Content-Type: application/json" \
+      -d '{"messages": [{"role": "user", "content": "Can you write a limerick about orange flavored popsicles?"}]}'
 
 * ``Embeddings``
 
 .. code-block:: bash
 
-    curl -X POST http://my.gateway:8888/gateway/embeddings/invocations -H "Content-Type: application/json" -d '{"texts": ["I'd like to return my shipment of beanie babies, please", "Can I please speak to a human now?"]}'
+    curl -X POST http://my.gateway:8888/api/2.0/gateway/gateway/embeddings/invocations \
```
Comment (dbczumar) [inline docs/source/gateway/index.rst:910]: Looks like gateway is duplicated

Comment (dbczumar) [inline docs/source/gateway/index.rst:910]: for invocations, we shouldn't have `/api/2.0`. We should only have `/api/2.0` for CRUD

**Review by dbczumar — APPROVED** [review]
LGTM pending `/gateway` addition to invocations! Thanks @harupy !

**Comment by harupy** [comment]
Looks better now:

image

**Pull request merged**